### ISSUE TITLE
[Bug] Fix edge case where user data from the database table doesn't write to local storage

### ISSUE THREAD

**Issue by mddilley** [cityofaustin/atd-data-tech#10019]
There is an edge case bug with the user auth frontend code where sign in creates an endless loop because `atd_moped_user_db_data` in local storage never writes. As a workaround, you can just copy the values from this entry staging local storage.

_Moped test instances are currently still in development so it will be more straightforward to fix this after we have a GitHub action to spin up test instances_

To reproduce:
1. Try to log into a fresh test with your staging username and password
2. You might get an endless loop where the Project Summary table never loads
3. If you take the value of `atd_moped_user_db_data` from the staging URL local storage and paste it into the `atd_moped_user_db_data` entry of the test instance, you will see the loop stop and the site will load

**Comment by johnclary** [comment]
@mddilley can you hazard a solution / estimate for this? Or, alternatively, shall I add it to the grooming agenda?

**Comment by mddilley** [comment]
@johnclary Yes, this would be good for grooming because I haven't seen this happen since we onboarded Rebecca so it would be good to know if this is still a bug. Doesn't seem to be an issue with test instances at all. Could be icebox-able?
